etomica-apps/src/main/java/etomica/spin/heisenberg/Heisenberg.java

```diff
@@ -60,7 +60,11 @@
 
     
     /**
-     * 
+     * 2D heisenberg model in square lattice.
```

Comment:
Javadoc requires a separate `@param` annotation for each parameter. So it would be like this:

```java
/**
 * @param _space 
 * @param nCells the total number of atoms
 * @param temperature 
 * @param interactionS the J in heisenberg energy function U = J*Cos(theta1-theta2)
 * ... and so on
 */
```